locales/en/messages.json

```
@@ -1183,20 +1183,29 @@
     "initialSetupInfoBuildConfig": {
         "message": "Config"
     },
+    "initialSetupInfoBuildDownload": {
+        "message": "Download"
+    },
+    "initialSetupInfoBuildFirmware": {
+        "message": "Firmware:"
+    },
     "initialSetupInfoBuildLog": {
         "message": "Log"
     },
     "initialSetupInfoBuildOptions": {
         "message": "Options"
     },
+    "initialSetupInfoBuildOptionsLabel": {
+        "message": "Options:"
```

We can re-use translation here:
```suggestion
        "message": "$t(initialSetupInfoBuildOptions.message):"
```

---

Obsolete; no longer exists in the refined design